css-text-3/Overview.bs

```diff
@@ -724,17 +724,23 @@ Characters and Letters</h3>
         (ignoring any intervening <a>inline box</a> boundaries)
         is removed.
       <li>If spaces or tabs at the end of a line are non-collapsible
-         but have 'white-space' set to ''pre-wrap''
-         the UA must either <a>hang</a> the <a>white space</a>
-         or visually collapse the character advance widths
-         of any overflowing spaces
-         such that they don't take up space in the line.
-         However, if 'overflow-wrap' is set to ''break-spaces'',
-         collapsing their advance width is not allowed,
-         as this would prevent the preserved spaces from wrapping.
-
-         Note: Hanging the white space rather than collapsing it
-         allows users to see the space when selecting or editing text.
+         but have 'white-space' set to ''pre-wrap'':
+	 <ul>
+	   <li>If the value of 'overflow-wrap' is ''overflow-wrap/normal'' or ''overflow-wrap/break-word''
+	     (i.e. it does not include ''break-spaces'')
+             the UA must either <a>hang</a> the <a>white space</a>
+             or visually collapse the character advance widths
+             of any overflowing spaces
+             such that they don't take up space in the line.
+
+             Note: Hanging the white space rather than collapsing it
+             allows users to see the space when selecting or editing text.
+	   <li>Otherwise
+	     (i.e. 'overflow-wrap' property includes ''break-spaces''),
+             hanging or collapsing the advance width of is not allowed.
+             The end-of-lines spaces that would overflow the line are
+             wrapped as described in the definition of the ''break-spaces'' value.
```

Comment:
Normatively, this doesn't belong here. We've already broken lines. If spaces needed to get wrapped, they've already gotten wrapped.